tdrs-frontend/src/components/SubmissionHistory/CaseAggregatesTable.jsx

```diff
@@ -26,13 +27,13 @@ const MonthSubRow = ({ data }) =>
 
 const CaseAggregatesRow = ({ file }) => {
   const dispatch = useDispatch()
-
+  const reprocessedOn = formatDate(getReprocessedDate(file))
   return (
     <>
       <tr>
         <th scope="rowgroup" rowSpan={3}>
           {formatDate(file.createdAt) + ' by ' + file.submittedBy}
-          {hasReparsed(file) && <></>}
+          {hasReparsed(file) && <ReprocessedModal date={reprocessedOn} />}
```

Comment:
So this doesn't break anything because of how the modals css/javascript work under the hood, but it is technically against DOM structure best practices to have an element appear visually "on top" but be nested inside other elements on the page. What you have would be how I would implement a tooltip component, but I'd recommend moving the modal to be a direct child of the fragment in `Reports.jsx` and letting the parent container own all the state, passing the `setState` function as a callback.

```jsx
export const Reports = () => {
  const [reprocessedModalVisible, setReprocessedModalVisible] = useState(false)
  // ......

  return (
    <>
      <div>
        ......
        <CaseAggregatesTable setReprocessedModalVisible={setReprocessedModalVisible} />
      </div>
      <Modal />
      <ReprocessedModal isVisible={reprocessedModalVisible} />
    </>
  )
}
```

`setReprocessedModalVisible` needs to be passed around a few times and may need to take some information like the reprocessed date (or the submission id, and the date can be fetched by the component). But would keep the page from creating a separate modal for each row. 

This line would then be changed to only contain the button, which executes the `setReprocessedModalVisible` callback

Reply:
Oh that is something I never considered. It did not occur to me that I was actually creating a modal per row in the tables. I will definitely make that change, thank you for the suggestion!

Reply:
@elipe17 @reitermb also while i'm thinking of it, tooltip might be available to us again after the v3 upgrade
https://designsystem.digital.gov/components/tooltip/